native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/DefaultStatementParameterProcessor.java

```diff
@@ -755,6 +757,7 @@ private void setRefAndStruct(Connection connection, PreparedStatement preparedSt
 
     private void setDateTimeAndTimestamp(PreparedStatement preparedStatement, String sqlType, int index, Object value)
             throws SQLException, DataError {
+        BMap zoneMap = null;
```

Comment:
```suggestion
        BMap zoneMap;
```

Reply:
We need to initialize this value. Otherwise, we will get the error.